draft-ietf-quic-multipath.md

```diff
@@ -1081,15 +1081,15 @@ This is a major difference from
 {{QUIC-TRANSPORT}}, which only defines three number spaces (Initial,
 Handshake and Application packets).
 
-For any given path, connection ID rotation, NAT Rebinding or path migration,
+For any given path, connection ID rotation, NAT rebinding, or client initiated migration
 as specified in {{QUIC-TRANSPORT}} might occur, similar as on a single path.
-These events do not change the path ID,  and do not affect the packet number
+These events do not change the Path ID,  and do not affect the packet number
 space associated with the path.
 
-In case of network-based NAT rebindings path migration cannot be avoided.
-However otherwise, as a general rule, when the multipath extension is supported,
-implementations  should strongly prefer opening a new path instead of path migration 
-as specified in {{QUIC-TRANSPORT}}.
+In case of network-based NAT rebindings, path migration cannot be avoided.
+However, as a general rule, when the multipath extension is supported,
+implementations  should otherwise strongly prefer opening a new path instead
+of path migration  as specified in {{QUIC-TRANSPORT}}.
```

Comment:
It might be better to specify the rule first, and the exceptions later. Something like:

It is generally preferable to use multipath mechanisms such as creating a new path and later abandoning the old path, rather than doing migration of a single path as specified in {{QUIC-TRANSPORT}}. This enables a smoother handover and allows a more controlled migration handling at the server side. However, migration of a single path cannot be
avoided in case of NAT rebinding, or if the server requests migration to a "preferred address" during the handshake.